### DISCUSSION TITLE
Training Existing Model(sm)

### DISCUSSION THREAD

**Discussion by sandeepreddy5** [explosion/spaCy#7828] [Help: Coding & Implementations] [answered]
I am using en_core_web_sm model for NER in my project and its actually working fine. But I need to add some new labels to the existing ner. So I gathered some training data and started training the existing model using CLI in spacy v3. Training completed successfully but the thing is I am unable to identify existing entities and it is only detecting what i trained only. Help me with this issue.
Thank in advance

**Comment by polm** [comment, marked as answer]
This problem is "catastrophic forgetting" and it's pretty common. We have a technique, outlined in [a blog post](https://explosion.ai/blog/pseudo-rehearsal-catastrophic-forgetting), that should help work around it, but the implementation is only experimental at this point. 

One simple workaround you can try is to include both the default NER component from `en_core_web_sm` and your NER component in the same pipeline.
